front-end/src/services/auth.service.tsx

```diff
@@ -35,8 +35,15 @@ export const isLocalStorageTokenValidOrUndefined = (): boolean => {
 	let token = localStorage.getItem('Authorization') || null;
 
 	if (token) {
-		const tokenPayload = jwt.decode(token) as JWTPayploadType;
-		return tokenPayload.exp > Date.now() / 1000
+		const tokenPayload = jwt.decode(token) as JWTPayploadType | null;
+
+		if(tokenPayload){
+			return tokenPayload.exp > Date.now() / 1000
```

Comment:
can be converted to 
tokenPayload ? tokenPayload.exp > Date.now() / 1000 : false

Reply:
I usually prefer when things are super clear even if it takes 4 more loc, but this one is super simple so I did condens it and kept the comment.